chart/values.schema.json

```
@@ -7409,6 +7409,14 @@
                             ],
                             "default": null
                         },
+                        "gitSshKey": {
```

```suggestion
                        "sshKey": {
```

I think maybe `sshKey` is a better name for this?

---

changed 👍